### ISSUE TITLE
[WP CLI] sample-data/sample_products.xml Failed to import taxonomy

### ISSUE THREAD

**Issue by kkmuffme** [woothemes/woocommerce#52358]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

sample-data/sample_products.xml seems to be invalid - it worked fine in previous versions (I did not test when exactly it broke)

### Expected behavior

No error

### Actual behavior

>Failed to import pa_color Blue: Invalid taxonomy.<br />Failed to import pa_color Green: Invalid taxonomy.<br />Failed to import pa_size Large: Invalid taxonomy.<br />Failed to import pa_size Medium: Invalid taxonomy.<br />Failed to import pa_color Red: Invalid taxonomy.<br />Failed to import pa_size Small: Invalid taxonomy.<br />Failed to import pa_color Blue: Invalid taxonomy.<br />Failed to import pa_color Green: Invalid taxonomy.<br />Failed to import pa_color Red: Invalid taxonomy.<br />Failed to import pa_color Blue: Invalid taxonomy.<br />Failed to import pa_color Gray: Invalid taxonomy.<br />Failed to import pa_color Red: Invalid taxonomy.<br />Failed to import pa_color Yellow: Invalid taxonomy.<br />Failed to import pa_color Gray: Invalid taxonomy.<br />Failed to import pa_color Green: Invalid taxonomy.<br />Failed to import pa_color Blue: Invalid taxonomy.<br />Failed to import pa_color Gray: Invalid taxonomy.<br />Failed to import pa_color Red: Invalid taxonomy.<br />

### Steps to reproduce

>wp import wp-content/plugins/woocommerce/sample-data/sample_products.xml --authors=create --quiet

### WordPress Environment

WC 9.4
WP 6.7 RC3

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by rodelgc** [comment]
Seems like the issue happens when import was done via WP CLI. Importing of the xml file worked for me when I tried it via the WordPress Importer.

**Comment by kkmuffme** [comment]
Yes exactly, happens in CLI - which is kind of the point of that data as it's mostly (probably only, with some exceptions) used for CI - which makes me wonder, why this won't make woocommerce's own CI fail
